docs/source/using_doctr/using_models.rst

```diff
@@ -330,14 +330,18 @@ For reference, here is the JSON export for the same `Document` as above::
     ]
   }
 
-To export the outpout as XML (hocr-format) you can use the `export_as_xml` method::
+To export the outpout as XML (hocr-format) you can use the `export_as_xml` method:
+
+.. code-block:: python
 
   xml_output = result.export_as_xml()
   for output in xml_output:
       xml_bytes_string = output[0]
       xml_element = output[1]
 
-For reference, here is a sample XML byte string output::
+For reference, here is a sample XML byte string output:
+
+.. code-block:: xml
```

Comment:
Does this render correctly ?

Reply:
Yep, but maybe it's due to `make html`:
![image](https://github.com/mindee/doctr/assets/43931730/524af953-d5b7-4596-9420-d7197f4ac94c)

Reply:
Nice 👍